src/components/DashboardTable/DashboardTable.module.scss

```
@@ -2,14 +2,22 @@
 
 .DashboardTable {
     width: 100%;
+    height: 100%;
+
+    overflow: hidden;
```

of op auto zetten, dan kan je er nog door scrollen
```suggestion
    overflow: auto;
```

---

Die overflow mag eig zelfs weg. De scrollbar verschijnt nu in `<TableContainer>` en maakt deze rule overbodig.